src/AltinnCore/Common/Languages/ini/en.ini

```diff
@@ -197,6 +197,14 @@ modal_properties_header = Edit properties
 modal_properties_add_check_box_options = How would you like to add check boxes?
 modal_properties_add_radio_button_options = How would you like to add radio buttons?
 modal_properties_trigger_validation_label = Skal feltet trigge en validering?
+modal_properties_file_upload_simple = Simple (Expects one attachement)
```

Comment:
The keys should be in alphabetical order

Reply:
Agreed. Sorted all keys in nb. and en.-ini files in alphabetical order.